dev/TabView/TabView.xaml

```diff
@@ -194,7 +194,6 @@
                                 Delay="50"
                                 FontFamily="{ThemeResource SymbolThemeFontFamily}"
                                 Interval="100"
-                                Style="{StaticResource RepeatButtonRevealStyle}"
```

Comment:
what does this look like now ?

Reply:
With the change:
![image](https://user-images.githubusercontent.com/21356912/108908337-0576f580-75d8-11eb-83ce-671a6e093a26.png)
Without the change:
![image](https://user-images.githubusercontent.com/21356912/108908388-1889c580-75d8-11eb-9a2e-e3c74d149c98.png)
TabView will be update soon:)

Reply:
@YuliKl @stmoy  Is this okay for now until we fix TabView?

Reply:
What work needs to be done for TabView? Leon is investigating drop shadows for TabViews right now, and wondering if future visual updates will regress these fixes.

Reply:
This seems okay for now since we don't have the finalized design for these arrows.